src/Bicep.Core.IntegrationTests/Bicep.Core.IntegrationTests.csproj

```diff
@@ -1,4 +1,4 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
nit: don't see the change

Reply:
I think it's either a space or encoding, in not sure how or why it happened but it's unintentional  and likely has no effect so let's ignore it